gloo/docs/installation/quick_start.md

```diff
@@ -22,6 +27,25 @@ Verify the CLI is installed and running correctly with:
 
 `glooctl --version`
 
+Now run:
+
+`glooctl install gateway`
+
+
+### Kubernetes manifest
+
+To 
+
+Use kubectl to install gloo using its released Kubernetes manifest:
+```
+export LATEST_RELEASE=$(curl -s "https://api.github.com/repos/solo-io/gloo/releases/latest" \
+| grep tag_name \
+| sed -E 's/.*"([^"]+)".*/\1/' )
+
+kubectl apply -f https://github.com/solo-io/gloo/releases/download/$LATEST_RELEASE/gloo-gateway.yaml
```

Comment:
i believe the commandline has (or will have after https://github.com/solo-io/gloo/pull/451) a dry run option that prints out the yaml. 
do you want to use that, is this the option for people who dont want to use the command line?

Reply:
Yah, it's for people who want to install directly from yaml and without the cli.